db/schema.rb

```diff
@@ -50,6 +50,10 @@
     t.datetime "created_at",                             :null => false
     t.datetime "updated_at",                             :null => false
     t.string   "role"
+    t.string   "name"
+    t.string   "city"
+    t.string   "country"
```

Comment:
Let's keep it. It's not a lot, and it would be cool to see where people are using the app from anyway.